.eslintrc.json

```diff
@@ -1,3 +1,51 @@
 {
-  "extends": "./script/node_modules/eslint-config-standard/eslintrc.json"
-}
+  "extends": [
+    "./script/node_modules/eslint-config-standard/eslintrc.json",
+    "./script/node_modules/eslint-config-prettier/index.js",
+    "./script/node_modules/eslint-config-prettier/standard.js"
+  ],
+  "plugins": [
+    "prettier"
+  ],
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 8,
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "globals": {
+    "atom": true,
+    "snapshotResult": true
+  },
+  "rules": {
+    "standard/no-callback-literal": ["off"],
+    "no-mixed-operators": ["off"],
+    "no-useless-escape": ["off"],
+    "no-return-await": ["off"],
+    "node/no-deprecated-api": ["off"],
+    "prefer-promise-reject-errors": ["off"],
+    "no-unused-expressions": ["off"],
+    "symbol-description": ["off"],
+    "no-use-before-define": ["off"],
+    "prettier/prettier": ["off"] // disable prettier rules for now.
+  },
+  "overrides": [
+    {
+      "files": ["spec/**", "**-spec.js", "**.test.js"],
+      "env": {
+        "jasmine": true
+      },
+      "globals": {
+        "advanceClock": true,
```

Comment:
Ideally these should be defined as a separate `.eslintrc.js` in the spec folder, but the bundled package specs mean it makes _some_ sense to define them here. If possible though I would recommend a tighter definition.

The same applies to the `jasmine` environment enabling.

Reply:
What do you mean by tighter? This is currently only applied to files that match the following globs: `["spec/**", "**-spec.js", "**.test.js"]`, which are basically tests

Reply:
Whoops, missed that!